src/modules/apiRequest.js

```diff
@@ -0,0 +1,50 @@
+const createNewGame = async (monopoly) => {
+  const request = new Request('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games');
+  await fetch(request, {
+    method: 'POST',
+    body: JSON.stringify({
+      name: monopoly,
+    }),
+    headers: {
+      'Content-Type': 'application/json; charset=UTF-8',
+    },
+  });
+};
+
+const postPlayer = async (playerName, playerScores) => {
+  const request = new Request('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/rGp2reoqk4GMj6QaC1QY/scores/');
+  await fetch(request, {
+    method: 'POST',
+    body: JSON.stringify({
+      username: playerName,
+      score: playerScores,
+    }),
+    headers: {
+      'Content-Type': 'application/json; charset=UTF-8',
+    },
+  });
+};
+
```

Comment:
- [ ] Your app does not work as required. When the app is run, a new game is not created and when a score is added it is not added to the API either. Kindly fix this.

- [ ] When I click on the refresh button nothing is gotten from the API either.

  ![image](https://github.com/zhorabay/Leaderboard/assets/54848363/8ef43efe-7f32-4a94-ae1a-591f1d225c1d)

Reply:
@DammyShittu I submitted the work to ask help, can you help me to fix this? I think I did everything correct but it shows error 400. Can you tell me what exactly is my mistake? what should i change? I know I can't add the score but why?